internal/seclang/rule_parser.go

```diff
@@ -447,9 +447,9 @@ actionLoop:
 				// skip unquoted whitespaces
 				continue actionLoop
 			}
-			cval.WriteString(string(c))
+			cval.WriteRune(c)
```

Comment:
Looks good. But this change is unrelated to zerolog and was added in this commit -> https://github.com/manojgop/coraza/pull/2/commits/187daf6cf954f3d79feaf35c18e8e9a12e549e59